stacks/ls-k8s-webadc/deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+#set -x
+
+################################################################################
+# repo
+################################################################################
+helm repo add ls-k8s-webadc https://litespeedtech.github.io/helm-chart
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="ls-k8s-webadc"
+CHART="ls-k8s-webadc/ls-k8s-webadc"
+CHART_VERSION="0.1.15"
+NAMESPACE="ls-k8s-webadc"
+
+kubectl get secret ls-k8s-webadc -n ls-k8s-webadc > /dev/null 2>&1
+if [ "$?" -ne "0" ]; then
```

Comment:
Could you simplify this logic a bit? Assume that the system installing it won't have a trail.key, serial.no, or license.key file.

Reply:
Removed the code which tested for a license and went on to create the trial license.